sites/public/pages/listings.tsx

```diff
@@ -272,11 +278,15 @@ const ListingsPage = () => {
               controlClassName="control"
               options={seniorHousingOptions}
             />
-            <FieldGroup
-              type="checkbox"
+            <Field
+              id="true"
```

Comment:
Could you change the id to something like `includeNulls`?

Reply:
Open to ideas here but when I change the id to anything but true, it doesn't set the includeNulls flag correctly and the filter stops working.